src/Components/Categories/CategoriesTable.js

```diff
@@ -0,0 +1,144 @@
+import { format as formatDate, parseISO } from "date-fns";
+import React, { useMemo, useState } from "react";
+import { useTable } from "react-table";
+import "./CategoriesTable.css";
+
+//fontawesome
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Modal from "./../Elements/Modal";
+import CategoryEditForm from "./../Categories/CategoryEditForm";
+import {
+  updateCategory,
+  deleteCategory,
+} from "../../Services/Organizations/CategoriesService";
+import { useCurrentOrganizationContext } from "../../Contexts/currentOrganizationContext";
+
+library.add(faTrashAlt);
+library.add(faEdit);
+
+export default function CategoriesTable(props) {
+  const renderDateColumn = (dateString) =>
+    formatDate(parseISO(dateString), "PP");
```

Comment:
I think we're using this in the other table(s), would you like to move this to the helpers?

Reply:
That sounds very wise.